content/blog/2021/01/camel-quarkus-release-1.6.0/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Camel Quarkus 1.6.0 Released"
+date: 2021-01-18
+authors: ["ppalaga"]
+categories: ["Releases", "Camel Quarkus"]
+preview: "Camel Quarkus 1.6.0 Released"
+summary: "The highlights of Camel Quarkus 1.6.0"
+---
+
+<sub><sup>Original image by <a href="https://commons.wikimedia.org/wiki/User:99of9">Toby Hudson</a> <a href="https://creativecommons.org/licenses/by-sa/3.0">CC BY-SA 3.0</a> via <a href="https://en.wikipedia.org/wiki/Camel_racing#/media/File:CamelRacingCamelCup2009Heat.JPG">Wikipedia</a></sup></sub>
+
+We are pleased to announce the release 1.6.0 of Camel Quarkus! Here are the highlights.
+
+## Java 11 required
+
+Having deprecated Java 8 [long time ago](https://camel.apache.org/blog/2020/04/Camel-Quarkus-release-1.0.0-M7/) and
```

Comment:
Any reason to link on the camel website directly ?
I'm curious whether "/blog/2020/04/Camel-Quarkus-release-1.0.0-M7/" would also work ?

Reply:
Let me try

Reply:
yes it does :+1: